### ISSUE TITLE
Make the channels sortable

### ISSUE THREAD

**Issue by pocketprotector** [psychs/limechat#292]
Hi please make the channel names sortable by alphabetical order. It is very hard to find a channel because they are randomly ordered.

**Comment by Cpuroast** [comment]
You do know that you can manually re-order them in whatever order you want and the order will be preserved.

**Comment by pvtmert** [comment]
please add 'sort by name' :)

**Comment by pvtmert** [comment]
i've made it available on #302, hopefully gets merged :)

**Comment by sup** [comment]
Yes please! TFW you have 30 channels that have slowly gotten out of alphabetical order and now have to manually re-order them :( Would be a super nice QOL improvement!
